gallery/src/components/NxStatefulToggle/NxStatefulToggleExample.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxStatefulToggle } from '@sonatype/react-shared-components';
+
+function NxStatefulToggleExample() {
+  const onChange = (checkedState: boolean) => { alert(checkedState); };
+
+  return (
+    <>
+      <fieldset className="nx-fieldset">
```

Comment:
Do toggles need to be in a fieldset?

Reply:
> Do toggles need to be in a fieldset?

No I don't think so. I've changed them to be in `form-group`s for now, but I wouldn't view that as a requirement. https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/ebc6e803a5c3c2bfa6280f36063c2c258eaadcf0